caluma/caluma_form/migrations/0033_auto_20200221_1346.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 2.2.10 on 2020-02-21 13:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('caluma_form', '0032_auto_20200220_1311'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='form',
+            name='slug',
+            field=models.SlugField(max_length=120, primary_key=True, serialize=False),
+        ),
+        migrations.AlterField(
+            model_name='historicalform',
+            name='slug',
+            field=models.SlugField(max_length=120),
+        ),
+        migrations.AlterField(
+            model_name='historicaloption',
+            name='slug',
+            field=models.SlugField(max_length=120),
+        ),
+        migrations.AlterField(
+            model_name='historicalquestion',
+            name='slug',
+            field=models.SlugField(max_length=120),
+        ),
+        migrations.AlterField(
+            model_name='option',
+            name='slug',
+            field=models.SlugField(max_length=120, primary_key=True, serialize=False),
+        ),
+        migrations.AlterField(
+            model_name='question',
+            name='slug',
+            field=models.SlugField(max_length=120, primary_key=True, serialize=False),
+        ),
+
+        # Manual alterations.
+        # https://code.djangoproject.com/ticket/25012
+        migrations.AlterField(
+            model_name='formquestion',
+            name='form',
+            field=models.ForeignKey('Form', on_delete=models.CASCADE),
+        ),
+        migrations.AlterField(
+            model_name='formquestion',
+            name='question',
+            field=models.ForeignKey('Question', on_delete=models.CASCADE),
+        ),
+        migrations.AlterField(
+            model_name='questionoption',
+            name='question',
+            field=models.ForeignKey('Question', on_delete=models.CASCADE),
+        ),
+        migrations.AlterField(
+            model_name='questionoption',
+            name='option',
+            field=models.ForeignKey('Option', on_delete=models.CASCADE),
+        ),
+
+        # The previous migrations fail to update the length of question_id.
+        migrations.RunSQL("ALTER TABLE caluma_form_formquestion ALTER COLUMN question_id TYPE character varying(120);"),
```

Comment:
Shouldn't this be done for `caluma_form_questionoption` too?

Reply:
This was a hack that shouldn't be done anywhere.

The reason I did this was because the this field didn't update its length with the 'normal' migration.

Reply:
Maybe best to mark this PR as a "draft"? Then everyone knows that reviews are not really wanted yet.

Reply:
How about it is assigned to a specific developer?

Reply:
To me that carries a different meaning in this repo. Since caluma is used in many projects, reviews are often done by multiple people, even when not assigned.